docs/reference/migration/migrate_5_0/scripting.asciidoc

```diff
@@ -1,6 +1,54 @@
 [[breaking_50_scripting]]
 === Script related changes
 
+==== Switched Default Language to Painless
+
+The default scripting language for Elasticsearch is now Painless.  Painless is a language similar to Groovy, except
+faster and more secure.  Many Groovy scripts will be identitical to Painless scripts to help make the transition
```

Comment:
"Painless is a language similar to Groovy, except faster and more secure" -> "Painless is a custom-built language with syntax similar to Groovy designed to be fast as well as secure"